batch.go

```diff
@@ -243,14 +233,30 @@ func newLabelPool(faker *fake.Faker, cardinalities map[string]int) LabelPool {
 		"format": []string{"apache_common", "apache_combined", "apache_error", "rfc3164", "rfc5424", "json", "logfmt"}, // needs to match the available flog formats
 		"os":     []string{"darwin", "linux", "windows"},
 	}
+
+	num := 10
+	var nss []string
```

Comment:
no need to declare this. just add : in line 242
nss := generateValues(faker.BS, num)